modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -1,30 +1,38 @@
-= Creating an API instance on API Manager
+= Creating an API instance
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager 2.x, you can use instances of an API in multiple environments, and you can have multiple instances in the same environment. You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies, for example to the proxy, and throttling policies and security policies to the basic endpoint.
+In API Manager 2.x, you can use instances of an API in multiple environments. Additionally, you can also have multiple instances in the same environment. 
 
-After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them:
+You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies, for example to the proxy, and throttling policies and security policies to the basic endpoint.
 
-image::instances.png[]
+After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
 
-. Set up an API to manage from Exchange.
+To create a API instance:
+
+. Select *API Administration > Manage API* to create an API instance from Exchange.
 +
-For example, set up version 1.0 of an API named jsonplaceholder.
+If you attempt to create two API verions of the same name, an instance of the existing API is automatically created. For example, set up version 1.0 of an API named jsonplaceholder.:
```

Comment:
```suggestion
If you attempt to create two API versions with the same name, an instance of the existing API is automatically created. For example, to set up version 1.0 of an API named `jsonplaceholder`:
```

Reply:
I think that this text needs clarification: are you setting up two new APIs with the same name, or are you setting up one new version as a copy of the existing version, both with the same name?

Reply:
Agree that I don't completely understand.

Reply:
Removed the sentence!